src/hooks/useDailyStatistics/utils.ts

```diff
@@ -41,14 +45,17 @@ export const countTotalTransactionsAndByCategory = (
           category: key,
           quantityText: formatQuantityText(
             totalQuantityInCategory,
-            findItemByCategory?.quantity.unit || {
+            c13ntForUnit(findItemByCategory?.quantity.unit) || {
               type: "POSTFIX",
-              label: " qty",
+              label: ` ${i18ntWithValidator(
+                "checkoutSuccessScreen",
+                "quantity"
+              )}`,
             }
           ),
           descriptionAlert:
             findItemByCategory?.categoryType === "APPEAL"
-              ? "via appeal"
+              ? i18ntWithValidator("redemptionStats", "viaAppeal")
```

Comment:
we're importing `i18ntWithValidator` directly from the useTranslate hook, but we're passing in just `c13ntForUnit` through the params here.

is there a reason why it is inconsistent here?

Reply:
i18ntWithValidator is called directly from useTranslate.tsx as it is not a hook. it is only exposed as a method while c13ntForUnit relies on the useTranslate hook that cannot be extracted due to dependency on useContext() from CampaignConfigContext

Reply:
hmm, i think as mentioned above, do try to follow the other implementations thus far, as we shouldn't be importing a hook inside another hook (:

Reply:
hi @ilhaoni 
I've moved all the translation related code from utils to TransactionHistoryCard so that the TransactionHistoryCard will be responsible for performing the translation instead of utils.